app/cypress/integration/application-form-validation.spec.js

```diff
@@ -95,6 +95,24 @@ describe('When reviewing a submitted application as an analyst', () => {
       component: 'Admin Summary Card',
       variant: 'with errors'
     });
+
+    // Override Justification screenshots
+    cy.get('.override-accordion').happoScreenshot({
+      component: 'Override Justification',
+      variant: 'closed'
+    });
+    cy.get('.override-accordion > .btn').click();
+    cy.get('.override-accordion').happoScreenshot({
+      component: 'Override Justification',
+      variant: 'open'
+    });
```

Comment:
The component selected is a bit too deep in the tree, resulting in Happo missing some CSS context. You should use the `.alert .override-accordion` selector instead.